src/rajAJun20/BankSystem.java

```diff
@@ -0,0 +1,66 @@
+package rajAJun20;
+
+public class BankSystem {
+
+	int balance = 50000;
+	int creditCount = 0;
+	int debitCount = 0;
+	int printBalanceCount = 0;
+	static int creditCount1 = 0;
+	static int debitCount1 = 0;
+	static int printBalanceCount1 = 0;
```

Comment:
Remove the initialization of all instance variables.

Reply:
Changes done...!!